docker/Dockerfile.php80

```diff
@@ -36,7 +36,8 @@ RUN apt update && apt-get install -y --no-install-recommends \
 RUN wget -O /usr/local/bin/phpunit8 https://phar.phpunit.de/phpunit-8.phar \
     && chmod +x /usr/local/bin/phpunit8 \
     && wget -O /usr/local/bin/phpunit9 https://phar.phpunit.de/phpunit-9.phar \
-    && chmod +x /usr/local/bin/phpunit9
+    && chmod +x /usr/local/bin/phpunit9 \
+    && ln -s /usr/local/bin/phpunit9 /usr/local/bin/phpunit
```

Comment:
Good idea. Can you update this in https://github.com/juliushaertl/nextcloud-docker-dev/blob/master/docker/Dockerfile.php.template#L39 and run https://github.com/juliushaertl/nextcloud-docker-dev/blob/master/Makefile#L49 to apply this to all dockerfiles and also keep it in place for future updates?

Reply:
Oh very nice, I didn't notice the template! 👍 
Done.